ivy/functional/frontends/torch/Tensor.py

```diff
@@ -145,6 +145,9 @@ def new_tensor(
     def view_as(self, other):
         return self.view(other.shape)
 
+    def detach(self):
+        return self.new_tensor(self.data, requires_grad=False)
```

Comment:
I think the ivy equivalent of torch's `detach` is the `stop_gradient` function, please use it with a `preserve_type=False` in order to get a tensor with `requires_grad=False`.

Reply:
Yeah but I think we could assume that the `torch.detach` function used inside the `stop_gradient` torch implementation should be more efficient with memory that our implementation.